.env.example

```
@@ -1,6 +1,10 @@
 APP_ENV=production
 APP_DEBUG=false
 APP_KEY=
-APP_URL=http://panel.test
+APP_URL=https://example.com
 APP_INSTALLED=false
 APP_LOCALE=en
+
+# Reverse Proxy Configuration
+TRUST_PROXIES=*
+SESSION_SECURE_COOKIE=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SESSION_SECURE_COOKIE key should go before the TRUST_PROXIES key

(UnorderedKey)